src/Network.cpp

```diff
@@ -942,7 +944,15 @@ std::vector<float> Network::gather_features(const GameState* const state,
                               symmetry);
     }
 
-    std::fill(to_move_it, to_move_it + NUM_INTERSECTIONS, float(true));
+    const auto black_to_move_it = begin(input_data) + 2 * INPUT_MOVES * NUM_INTERSECTIONS;
+    const auto white_to_move_it = black_to_move_it + NUM_INTERSECTIONS;
+    const auto pos_komi = get_normalised_komi(state);
+    const auto neg_komi = 1.0f - pos_komi;
+    const auto black_komi = blacks_move ? pos_komi : neg_komi;
+    const auto white_komi = blacks_move ? neg_komi : pos_komi;
```

Comment:
Conversely, having these variable names end with something like `_norm_komi` would be proper.